services/has-many-getter.js

```diff
@@ -32,51 +32,29 @@ function HasManyGetter(model, association, opts, params) {
     queryOptions.where = where;
     queryOptions.include = include;
 
-    return model.findById(params.recordId)
-      .then(function (record) {
-        return record['get' +
-          _.upperFirst(params.associationName)](queryOptions);
-      });
+    return model.findById(params.recordId, {
+      include: [{
+        model: association,
+        as: params.associationName,
+        scope: false,
+        where,
+        include,
+      }],
+    }).then(function (records) {
+      return records[params.associationName];
+    });
```

Comment:
Is it just a refactoring or does it fix something?

If it is a refactoring:
- I am not sure which version is simpler.
- queryOptions are not used anymore which seems dangerous (is the pagination still working for example)

If it is a fix, then let us know what it fixes in the changelog.

Reply:
This is cool to improve the performances (even tough 2 SQL queries vs 1 does not necessary mean performance improvement).
But the fact is that this function change generate other changes in other components that make the code less maintainable to me.

Another point is that, the `getCount` function fix seems almost good to me. This was the objective of the PR.
This change/refactoring is slowing down the fix release because I am not fan of the changes related to it.
As it is a different subject, I suggest you to move this refactoring in another PR/branch.

Reply:
I need to check if it works by rolling back. At first I did it because it was not working the old way